src/Tribe/Deprecation.php

```diff
@@ -117,8 +117,14 @@ public function deprecated_action_message() {
 
 	/**
 	 * Triggers a deprecation notice if there is any callback hooked on a deprecated filter.
+	 *
+	 * @since TBD the filtered value is passed through unchanged
+	 *
+	 * @param mixed $value
+	 *
+	 * @return mixed
 	 */
-	public function deprecated_filter_message() {
+	public function deprecated_filter_message( $value ) {
```

Comment:
Add a Default, to prevent problems when attaching with `0` as the params

Reply:
good call, done